SEOImages: User Story 1463880, Part 1 (#10786)
dotnet/docs: docs/framework/wpf/advanced/opentype-font-features.md

  
 The following text displays superscripts for the Palatino Linotype font.  
  

>  ![Text using OpenType superscripts](../../../../docs/framework/wpf/advanced/media/opentype-font-features/opentype-superscripts.gif "Text using OpenType superscripts")  

Reviewer: The migration tool wasn't very smart. We can simplify image paths a bit too.

```suggestion
 ![Text using OpenType superscripts](./media/opentype-font-features/opentype-superscripts.gif "Text using OpenType superscripts")  
```

> Text using OpenType superscripts  

Reviewer: Ok, 1st topic and we already run into unusual things. 😄 
This topic has a label under the image. So I'd remove this label from under the images, otherwise, screen readers will read this twice.
But I don't think this will be very common.

### Titling Capitals  
 Titling capitals are lighter in weight and proportion and designed to give a more elegant look than normal capitals. Titling capitals are typically used in larger font sizes as headings. The following text displays normal and titling capitals for the Pescadero font. Notice the narrower stem widths of the text on the second line.  
  

>  ![Text using OpenType titling capitals](./media/opentype-titling-capitals.gif "Text using OpenType titling capitals")  

Reviewer: can you fix the warnings? All the paths have to be changed to contain the new folder. For example:

```suggestion
 ![Text using OpenType titling capitals](./media/opentype-font-features/opentype-titling-capitals.gif "Text using OpenType titling capitals")  
```

Author: Yes. Fixed. Please confirm that things look good.

  
 [!code-xaml[OpenTypeFontSamples#4](~/samples/snippets/csharp/VS_Snippets_Wpf/OpenTypeFontSamples/CS/PageOne.xaml#4)]  
  
 The following text displays discretionary ligature glyphs for the Pericles font.  
  
 ![Text using OpenType discretionary ligatures](./media/opentype-font-features/opentype-discretionary-ligatures.gif "Text using OpenType discretionary ligatures")  
  
 The following markup example shows how to define discretionary ligature glyphs for the Pericles font, using properties of the <xref:System.Windows.Documents.Typography> object.  
  
 [!code-xaml[OpenTypeFontSamples#5](~/samples/snippets/csharp/VS_Snippets_Wpf/OpenTypeFontSamples/CS/PageOne.xaml#5)]  
  
 By default, [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] fonts in [!INCLUDE[TLA#tla_winclient](../../../../includes/tlasharptla-winclient-md.md)] enable standard ligatures. For example, if you use the Palatino Linotype font, the standard ligatures "fi", "ff", and "fl" appear as a combined character glyph. Notice that the pair of characters for each standard ligature touch each other.  
  

>  ![Text using OpenType standard ligatures](./media/opentype-font-features/opentype-standard-ligatures-palatino.gif "Text using OpenType standard ligatures with Palatino Linotype")    

Reviewer: ```suggestion
 ![Text using OpenType standard ligatures with Palatino Linotype](./media/opentype-font-features/opentype-standard-ligatures-palatino.gif "Text using OpenType standard ligatures with Palatino Linotype")    
```

Author: Updated based on suggestions.

  
 ![Text using OpenType standard glyphs](./media/opentype-font-features/opentype-standard-glyphs.gif "Text using OpenType standard glyphs")  
  
  
 The Pericles [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] font contains additional glyphs that provide stylistic alternates to the standard set of glyphs. The following text displays stylistic alternate glyphs.  
  
 ![Text using OpenType stylistic alternate glyphs](./media/opentype-font-features/opentype-stylistic-alternate-glyphs.gif "Text using OpenType stylistic alternate glyphs")  
  
 The following markup example shows how to define stylistic alternate glyphs for the Pericles font, using properties of the <xref:System.Windows.Documents.Typography> object.  
  
 [!code-xaml[OpenTypeFontSamples#2](~/samples/snippets/csharp/VS_Snippets_Wpf/OpenTypeFontSamples/CS/PageOne.xaml#2)]  
  
 The following text displays several other stylistic alternate glyphs for the Pericles font.  
  

>  ![Text using OpenType stylistic alternate glyphs](./media/opentype-font-features/opentype-stylistic-alternate-glyphs-periclese.gif "Text using OpenType stylistic alternate glyphs")  

Reviewer: we should rename the file name from Periclese -> Periclese. Also CATS is complaining about a dupe alt text so let's switch to something else.

```suggestion
 ![Text using OpenType stylistic alternate glyphs  for the Pericles font](./media/opentype-font-features/opentype-stylistic-alternate-glyphs-pericles.gif "Text using OpenType stylistic alternate glyphs  for the Pericles font")
```

Author: Updated file name and alt text based on suggestions.

## Alternates  
 Alternates are glyphs that can be substituted for a standard glyph. [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] fonts, such as the Pericles font used in the following examples, can contain alternate glyphs that you can use to create different appearances for text. The following text displays standard glyphs for the Pericles font.  
  
 ![Text using OpenType standard glyphs](./media/opentype-standard-glyphs.gif "Text using OpenType standard glyphs")  
  
  
 The Pericles [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] font contains additional glyphs that provide stylistic alternates to the standard set of glyphs. The following text displays stylistic alternate glyphs.  

>   

Reviewer: it seems that this one and another image were also being used by another topic. So let's create the copy as we discussed associated with that particular article.

Author: Images copied to directory for the additional article.
